model/post_list.go

```diff
@@ -95,6 +95,16 @@ func (o *PostList) AddPost(post *Post) {
 	o.Posts[post.Id] = post
 }
 
+func (o *PostList) ExtendAll(other *PostList) {
```

Comment:
Would it make sense to just change how `Extend` works instead of adding another function that works very similarly? `Extend` is only used in one other place than your code, so I don't think it would be too much of a problem to change it

Reply:
I did it on my first commit and changed after.  Yup, I agree; I combined those now.